Back-end/index.js

```diff
@@ -127,6 +140,31 @@ app.get("/user/:id", async (req, res) => {
   });
   res.json(user);
 });
+
+app.get("/comments/:id", async (req, res) => {
+  // Get Comments from post
+  const PostId = req.params.id;
+  const comments = await prisma.comments.findMany({
+    where: {
+      Post_id: parseInt(PostId),
+    },
+  });
+  res.json(comments);
+});
+
+app.post("/comment/:id", async (req, res) => {
+  const PostId = req.params.id;
+  const { userId, comment } = req.body;
+  const newComment = await prisma.comments.create({
+    data: {
+      userId: parseInt(userId),
```

Comment:
Suggest to use either camel case or the sneak case, but not mixed.

Reply:
I am a little confused on what you mean about camel/sneak case so I will ask you in person tomorrow.